server/db/db.go

```diff
@@ -1,13 +1,33 @@
 package db
 
 import (
+	"fmt"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/driver/postgres"
 	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 )
 
 var Instance *gorm.DB
 
-func New(path string) (err error) {
-	Instance, err = gorm.Open(sqlite.Open(path), &gorm.Config{})
+func New(driver, path string) (*gorm.DB, error) {
```

Comment:
Ich kenn mich bisher mit go nicht aus. Müsste der Klarheit halber der driver nicht als "driver string" in der Parameterdefinition angelegt werden?